src/sentry/filters/legacy_browsers.py

```diff
@@ -11,14 +14,53 @@
     'Safari': 6,
     'Edge': 0,
     'Opera': 15,
+    'Android': 4,
 }
 
 
+class LegacyBrowserFilterSerializer(serializers.Serializer):
+    # TODO: maybe find a serializer that handles multple values already
+    active = serializers.BooleanField()
+    subfilters = MultipleChoiceField(choices=[
+        'ie8',
+        'ie9',
+        'opera',
+        'android',
+        'safari'
+    ])
+
+
 class LegacyBrowsersFilter(Filter):
     id = 'legacy-browsers'
     name = 'Filter out known errors from legacy browsers'
     description = 'Older browsers often give less accurate information, and while they may report valid issues, the context to understand them is incorrect or missing.'
     default = False
+    serializer_cls = LegacyBrowserFilterSerializer
+
+    def is_enabled(self):
```

Comment:
We discussed this offline.

This `is_enabled()` value is used by the API endpoint in the `project_filters` endpoint. So it's being overloaded to mean something slightly different here. But this difference in behavior is worth documenting rather than changing all the APIs related to better suit this for now.

Though, this code does, I think, break existing behavior if a `1` or `0` is stored currently, then the API will get the string and not a boolean, so probably need to change code to something like:

```python
rv = ProjectOption.objects.get_value(
    project=self.project,
    key='filters:{}'.format(self.id),
    default='1' if self.default else '0',
)

if rv == '1':
    return True
if rv == '0':
    return False

return rv
```